lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -3032,7 +3040,7 @@ namespace Js
                             }
                             else
                             {
-                                SetArrayLikeObjects(pDestObj, idxDest, subItem);
+                                ThrowErrorOnFailure(SetArrayLikeObjects(pDestObj, idxDest, subItem), scriptContext, idxSubItem);
```

Comment:
It should be idxDest, nice catch. will fix that.

---
In reply to: [87446064](https://github.com/Microsoft/ChakraCore/pull/1969#discussion_r87446064) [](ancestors = 87446064)